docs/lib/storage/fragments/flutter/upload.md

```diff
@@ -0,0 +1,13 @@
+To upload to S3 from a data object, specify the key and the file to be uploaded. 
+
+```dart
+try {
+  UploadFileResult result = await Amplify.Storage.uploadFile(
+    key: key,
+    local: local,
```

Comment:
add this like to give an example of what local is:
File local = File('$path/filename.txt')

Reply:
done.